cdc/processor.go

```diff
@@ -580,6 +579,11 @@ func (p *processor) syncResolved(ctx context.Context) error {
 		pendingTxns = pendingTxns[:0]
 		return nil
 	}
+
+	defer func() {
+		close(p.executedTxns)
+	}()
```

Comment:
The two `defer` can be simplified to one-liners now?

Reply:
The style is not that important

Reply:
Yet open coded defer is a little faster